src/Repository/JobRepository.php

```diff
@@ -5,9 +5,27 @@
 namespace EMS\CoreBundle\Repository;
 
 use Doctrine\ORM\EntityRepository;
+use EMS\CoreBundle\Entity\Job;
 
 class JobRepository extends EntityRepository
 {
+    public function findById(int $jobId): Job
+    {
+        $qb = $this->createQueryBuilder('j');
+        $query = $qb
+            ->andWhere($qb->expr()->eq('j.id', ':job_id'))
+            ->setParameter('job_id', $jobId)
+            ->getQuery();
+
+        $job = $query->getSingleResult();
```

Comment:
As you search for by the id, find should works:
```suggestion
        $job = $this->find($jobId);
```

Reply:
aggree

Reply:
https://github.com/ems-project/EMSCoreBundle/pull/768/commits/74c4af8502ce1086e69f300ec66437053e548263